dbus_systemcalc.py

```diff
@@ -617,12 +631,14 @@ def _device_added(self, service, instance, do_service_change=True):
 
 		if do_service_change:
 			self._handleservicechange()
+		self._updatepvinverterspidlist()
```

Comment:
would be better to call _updatepvinverterspidlist() from within ._handleservicechange(). To keep it all together, and also you'll have a (small) performance improvement: the way it is done now, the code will loop through all pv inverters for every service in the list (see line 222).